lib/algoliasearch-rails.rb

```diff
@@ -821,7 +806,7 @@ def algolia_ensure_init(options = nil, settings = nil, index_settings = nil)
                    index_settings.delete(:slaves) ||
                    index_settings.delete('slaves')
         index_settings[used_slaves ? :slaves : :replicas] = replicas unless replicas.nil? || options[:inherit]
```

Comment:
These lines and the lines above still refer to the old `:slaves` setting, can you remove these in favor of `replicas`, please? 🙂